File: sort/bubble/bubblesort.js
function swap(arr,i,j){
    let tmp = arr[i]
    arr[i]=arr[j]
    arr[j]=tmp
    return arr
}

function bubbleSort(arr, n){
    let i,j
    for(i =0;i<n;i++){
        let swapped = false
        for(j=0;j<n;j++){
            if(arr[j]>arr[j+1]){
                arr = swap(arr,j,j+1)
                swapped = true
            }
        }
        if(swapped==false)
            break
    }

    return arr
}

const arr = [1,3,2,5,3,7,6,4]
const sorted = bubbleSort(arr,arr.length)
console.log(sorted)

File: sortAlgos/bubbleSort/bubblesort.py
def bubblesort(arr,n):
    for i in range(n):
        swapped = False
        for j in range(n):
            if(j<n-1 and arr[j]>arr[j+1]):
                arr[j], arr[j+1] = arr[j+1],arr[j]
                swapped = True
        if(swapped is False):
            break
    return arr


array = [1,2,4,7,5,3,6]
sortedarr = bubblesort(array,len(array))
print(sortedarr)
                